flow/layers/image_filter_layer.cc

```diff
@@ -103,6 +104,18 @@ void ImageFilterLayer::Paint(PaintContext& context) const {
   AutoCachePaint cache_paint(context);
 
   if (context.raster_cache) {
+    // Special case an ImageFilter.matrix transformation that only
+    // applies scaling. This is used by the default Android zoom page
+    // transition.
+    auto matrix_filter = filter_->asMatrix();
+    if (matrix_filter != nullptr &&
+        matrix_filter->matrix().isScaleTranslate() &&
```

Comment:
Why only scale translate? If you fall through because it is not a scale translate you are going to get a bitmap transform anyway.